kafka-ui-react-app/src/components/Dashboard/__test__/Dashboard.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Dashboard from 'components/Dashboard/Dashboard';
+
+const component = shallow(<Dashboard />);
+
+describe('Dashboard', () => {
+  it('section can be truthy', () => {
```

Comment:
"section can be truthy" what does it mean?

I would suggest to use something like `it renders section`

Reply:
done